UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Controllers/ProductDataController.cs

```diff
@@ -53,6 +53,7 @@ public class ProductDataController : BaseController<ProductDataController>
         /// </remarks>
         /// <param name="productIdentifiers">The JSON body containing product identifiers.</param>
         /// <param name="callbackUri">An optional callback URI that will be used to notify the requestor once the requested Exchange Set is ready to download from the File Share Service. The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same Exchange Set data as the response to the original API request. If not specified, then no call back notification will be sent.</param>
+        /// <param name="isUnencrypted">An optional isUnencrypted parameter will create encrypted Exchange set when value is passed as false and unencrypted Exchange Set when  value is passed as true.Bydefault value for isUnencrypted would always be false.</param>
```

Comment:
```suggestion
        /// <param name="isUnencrypted">An optional isUnencrypted parameter determines whether the Exchange set is encrypted or not. If the value is false, the Exchange set will be encrypted. If the value is true, the Exchange set will be unencrypted. The default value of isUnencrypted is false, which means the Exchange set is encrypted by default.</param>
```

Reply:
Update the suggested comment for other endpoints as well.

Reply:
Resolved